scripts/composer/AssetInstaller.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+/**
+ * @file
+ * Contains \DrupalProject\composer\AssetInstaller.
+ */
+
+namespace DrupalProject\composer;
+
+use Composer\Script\Event;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Finder\Finder;
+
+class AssetInstaller {
+
+  public static $assetFileTypes = [
+    '.htaccess',
+    '*.css',
+    '*.ico',
+    '*.jpeg',
+    '*.jpg',
+    '*.js',
+    '*.png',
+    '*.svg'
+  ];
+
+  public static $frontControllers = [
+    'index.php',
+    'core/install.php',
+    'core/rebuild.php',
+    'core/modules/statistics/statistics.php',
+  ];
+
+  public static function install(Event $event) {
+    $extra = $event->getComposer()->getPackage()->getExtra();
+    if (!isset($extra['drupal-app-dir'])) {
+      throw new \RuntimeException('Please configure drupal-app-dir in your composer.json');
+    }
+    if (!isset($extra['drupal-web-dir'])) {
+      throw new \RuntimeException('Please configure drupal-web-dir in your composer.json');
+    }
+    foreach (static::$frontControllers as $fileName) {
+      static::createStubPhpFile($extra['drupal-app-dir'], $extra['drupal-web-dir'], $fileName);
```

Comment:
It seems that `makePathRelative()` expects absolute paths to do the comparison. In this case it should be:
`static::createStubPhpFile(realpath($extra['drupal-app-dir']), realpath($extra['drupal-web-dir']), $fileName);`